kokoro/config/test/common.gcl

```diff
@@ -1,29 +1,41 @@
 import '../utils/functions.gcl' as functions
 
+// Note: if we ever need to change regions, we will need to set up a new
+// Cloud Router and Cloud NAT gateway for that region. This is because
+// we use --no-address on Kokoro, because of b/169084857.
+// The new Cloud NAT gateway must have "Minimum ports per VM instance"
+// set to 512 as per this article:
+// https://cloud.google.com/knowledge/kb/sles-unable-to-fetch-updates-when-behind-cloud-nat-000004450
+local _default_zones = join([
+  'us-central1-a=3',
+  'us-central1-b=3',
+  'us-central1-c=3',
+  'us-central1-f=3',
+  'us-east1-b=2',
+  'us-east1-c=2',
+  'us-east1-d=2',
+], ',')
+// T2A machines are only available on us-central1-{a,b,f}.
+// See warning above about changing regions.
+local _arm64_zones = join([
+  'us-central1-a',
+  'us-central1-b',
+  'us-central1-f',
+], ',')
+
 template config common = {
   params = {
     platforms = external
     build_file = external
 
+    // This is only used for determining the value of ZONES.
+    arch = 'unknown'
```

Comment:
Why not just default to `x86_64` here? That's really what we mean.

(And I would probably put the two zone lists in a parent tuple, and use `ZONES = _zones.get(arch)` to look up using arch as the key)

Reply:
OK done